pixart.js

```diff
@@ -1 +1,43 @@
+var brush = document.querySelector(".brush");
+var button = document.querySelector("#set-color");
+var colorInput = document.querySelector("#color-field");
+var color = "#1B4370";
 
+function startGame() {
+  button.addEventListener("click", setColorButton);
+  colorInput.addEventListener("keydown", enterColorInput);
+};
+
+enterColorInput = function() {
+  var color = colorInput.value;
+
+  brush.style.backgroundColor = color;
+};
+
+setColorButton = function() {
+  event.preventDefault();
```

Comment:
This function is doing the same thing as `enterColorInput `above. How could you refactor and only use one?